unit_tests/test_sct_events_operator.py

```diff
@@ -10,7 +10,7 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2020 ScyllaDB
-
+import datetime
```

Comment:
Changes to this file has already been merged here: https://github.com/scylladb/scylla-cluster-tests/pull/4283

Reply:
removed